src/test.js

```diff
@@ -377,16 +376,23 @@ Test.prototype = {
 			}
 		}
 
+		function moduleInModuleChain( testModule ) {
```

Comment:
Yes, I think it would be more clear as something like `moduleChainIdMatch` (with the other one being renamed to `moduleChainNameMatch`). We could also simplify the body:
```js
return inArray( testModule.moduleId, config.moduleId ) > -1 ||
	testModule.parentModule && moduleChainIdMatch( testModule.parentModule );
```